### ISSUE TITLE
Allow to specify working directory for the application

### ISSUE THREAD

**Issue by mgerhardy** [ValveSoftware/vogl#208]
The application that you want to create the trace file for might be built out of source - thus it would be cool to be able to specify the working directory in the launch dialog.

**Comment by haikarainen** [comment]
I agree, this is a very important yet basicfeature. I will take a look at the source and see if I can get a pull request going.
